vs-classes.cs

```diff
@@ -0,0 +1,4369 @@
+﻿//------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated by a tool.
+//     Runtime Version:4.0.30319.42000
+//
+//     Changes to this file may cause incorrect behavior and will be lost if
+//     the code is regenerated.
+// </auto-generated>
+//------------------------------------------------------------------------------
```

Comment:
As a rule I don't commit auto-generated files to a repository, but I will commit a makefile or build script that generates those files. Is this vs-classes file something that can be easily generated at build time? Is there a good reason to include this autogenerated file in the repo?

Reply:
The reason I don't commit autogenerated code is to discourage people from editing files. If the file is version controlled people are more likely to edit it and break it

Reply:
How does this code relate to `Classes.cs`? This is the xml parser right?

Reply:
Also, where is this file used?

Reply:
I'm just using it for reference file and editing `Classes.cs` so, really, people don't need a script to re-generate the file either. I added it to the gitignore, that seems reaosnable?

Reply:
Sounds good to me!

Reply:
Yeah I mean they do pop up in intellisense on all of the classes but it's not much of a bother